apps/modernization-ui/src/apps/search/advancedSearch/AdvancedSearch.tsx

```diff
@@ -527,16 +529,16 @@ export const AdvancedSearch = () => {
                                             onClick={() => setShowSorting(!showSorting)}
                                             outline>
                                             Sort by
-                                            <img
-                                                style={{ marginLeft: '5px' }}
-                                                src={
+                                            <IconComponent
```

Comment:
question (non-blocker): Would it make more sense to import this as `NBSIcon` or something like that to make it more evident that it's a non USWDS Icon ?